examples\graph\src\nodes.rs fails to build

On windows with recent rust(1.46) it fails to build

```
PS C:\Users\kev009\Documents\rust\qmetaobject-rs> cargo build
warning: `extern` fn uses type `std::cell::RefCell<dyn QObject>`, which is not FFI-safe
   --> qmetaobject\src\lib.rs:569:51
    |
569 | pub unsafe extern "C" fn RustObject_metaObject(p: *mut RefCell<dyn QObject>) -> *const QMetaObject {
    |                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^ not FFI-safe
    |
    = note: `#[warn(improper_ctypes_definitions)]` on by default
    = help: consider adding a `#[repr(C)]` or `#[repr(transparent)]` attribute to this struct
    = note: this struct has unspecified layout

warning: `extern` fn uses type `std::cell::RefCell<dyn QObject>`, which is not FFI-safe
   --> qmetaobject\src\lib.rs:575:49
    |
575 | pub unsafe extern "C" fn RustObject_destruct(p: *mut RefCell<dyn QObject>) {
    |                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^ not FFI-safe
    |
    = help: consider adding a `#[repr(C)]` or `#[repr(transparent)]` attribute to this struct
    = note: this struct has unspecified layout

warning: 2 warnings emitted

   Compiling graph v0.1.0 (C:\Users\kev009\Documents\rust\qmetaobject-rs\examples\graph)
error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:30:18
   |
30 |     cpp!(unsafe [s as "NoisyNode**", item_ptr as "QQuickItem*"] {
   |                  ^
   |                  |
   |                  unresolved import
   |                  help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:30:38
   |
30 |     cpp!(unsafe [s as "NoisyNode**", item_ptr as "QQuickItem*"] {
   |                                      ^^^^^^^^
   |                                      |
   |                                      unresolved import
   |                                      help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:30:5
   |
30 | /     cpp!(unsafe [s as "NoisyNode**", item_ptr as "QQuickItem*"] {
31 | |         if (!*s && item_ptr) {
32 | |             *s = new NoisyNode(item_ptr->window());
33 | |         }
34 | |     });
   | |       ^
   | |       |
   | |_______unresolved import
   |         help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:38:18
   |
38 |     cpp!(unsafe [s as "NoisyNode**", rect as "QRectF"] {
   |                  ^
   |                  |
   |                  unresolved import
   |                  help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:38:38
   |
38 |     cpp!(unsafe [s as "NoisyNode**", rect as "QRectF"] {
   |                                      ^^^^
   |                                      |
   |                                      unresolved import
   |                                      help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:38:5
   |
38 | /     cpp!(unsafe [s as "NoisyNode**", rect as "QRectF"] {
39 | |         if (*s) {
40 | |             (*s)->setRect(rect);
41 | |         }
42 | |     });
   | |       ^
   | |       |
   | |_______unresolved import
   |         help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:47:18
   |
47 |     cpp!(unsafe [s as "GridNode**", rect as "QRectF"] {
   |                  ^
   |                  |
   |                  unresolved import
   |                  help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:47:37
   |
47 |     cpp!(unsafe [s as "GridNode**", rect as "QRectF"] {
   |                                     ^^^^
   |                                     |
   |                                     unresolved import
   |                                     help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:47:5
   |
47 | /     cpp!(unsafe [s as "GridNode**", rect as "QRectF"] {
48 | |         if (!*s) *s = new GridNode;
49 | |         (*s)->setRect(rect);
50 | |     });
   | |       ^
   | |       |
   | |_______unresolved import
   |         help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:56:18
   |
56 |     cpp!(unsafe [s as "LineNode**", size as "float", spread as "float", color as "QColor"] {
   |                  ^
   |                  |
   |                  unresolved import
   |                  help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:56:37
   |
56 |     cpp!(unsafe [s as "LineNode**", size as "float", spread as "float", color as "QColor"] {
   |                                     ^^^^
   |                                     |
   |                                     unresolved import
   |                                     help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:56:54
   |
56 |     cpp!(unsafe [s as "LineNode**", size as "float", spread as "float", color as "QColor"] {
   |                                                      ^^^^^^
   |                                                      |
   |                                                      unresolved import
   |                                                      help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:56:73
   |
56 |     cpp!(unsafe [s as "LineNode**", size as "float", spread as "float", color as "QColor"] {
   |                                                                         ^^^^^
   |                                                                         |
   |                                                                         unresolved import
   |                                                                         help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:56:5
   |
56 | /     cpp!(unsafe [s as "LineNode**", size as "float", spread as "float", color as "QColor"] {
57 | |         if (!*s) *s = new LineNode(size, spread, color);
58 | |     });
   | |       ^
   | |       |
   | |_______unresolved import
   |         help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:64:18
   |
64 |     cpp!(unsafe [s as "LineNode**", rect as "QRectF", samples_ptr as "double*",
   |                  ^
   |                  |
   |                  unresolved import
   |                  help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:64:37
   |
64 |     cpp!(unsafe [s as "LineNode**", rect as "QRectF", samples_ptr as "double*",
   |                                     ^^^^
   |                                     |
   |                                     unresolved import
   |                                     help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:64:55
   |
64 |     cpp!(unsafe [s as "LineNode**", rect as "QRectF", samples_ptr as "double*",
   |                                                       ^^^^^^^^^^^
   |                                                       |
   |                                                       unresolved import
   |                                                       help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:65:18
   |
65 |                  samples_len as "std::size_t"] {
   |                  ^^^^^^^^^^^
   |                  |
   |                  unresolved import
   |                  help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\nodes.rs:64:5
   |
64 | /     cpp!(unsafe [s as "LineNode**", rect as "QRectF", samples_ptr as "double*",
65 | |                  samples_len as "std::size_t"] {
66 | |         if (!*s) return;
67 | |         QList<qreal> samples;
...  |
70 | |         (*s)->updateGeometry(rect, samples);
71 | |     });
   | |       ^
   | |       |
   | |_______unresolved import
   |         help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\main.rs:37:22
   |
37 |         cpp!(unsafe [obj as "QQuickItem *"] { obj->setFlag(QQuickItem::ItemHasContents); });
   |                      ^^^
   |                      |
   |                      unresolved import
   |                      help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: unresolved import
  --> examples\graph\src\main.rs:37:9
   |
37 |         cpp!(unsafe [obj as "QQuickItem *"] { obj->setFlag(QQuickItem::ItemHasContents); });
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |         |
   |         unresolved import
   |         help: a similar path exists: `lazy::core`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 21 previous errors

For more information about this error, try `rustc --explain E0433`.
error: could not compile `graph`.
```

--- a/examples/graph/Cargo.toml
+++ b/examples/graph/Cargo.toml
@@ -1,6 +1,7 @@
 [package]
 name = "graph"
 version = "0.1.0"
+edition = "2018"
 authors = ["Olivier Goffart <ogoffart@woboq.com>"]
 build = "build.rs"
 

--- a/examples/graph/src/nodes.rs
+++ b/examples/graph/src/nodes.rs
@@ -2,7 +2,7 @@ use qmetaobject::scenegraph::SGNode;
 use qmetaobject::{QColor, QQuickItem, QRectF};
 
 qrc! {
-    init_ressource,
+    init_resource,
     "scenegraph/graph" {
 //        "main.qml",
         "shaders/noisy.vsh",
@@ -25,7 +25,7 @@ cpp! {{
 pub enum NoisyNode {}
 
 pub fn create_noisy_node(s: &mut SGNode<NoisyNode>, ctx: &dyn QQuickItem) {
-    init_ressource();
+    init_resource();
     let item_ptr = ctx.get_cpp_object();
     cpp!(unsafe [s as "NoisyNode**", item_ptr as "QQuickItem*"] {
         if (!*s && item_ptr) {
@@ -52,7 +52,7 @@ pub fn update_grid_node(s: &mut SGNode<GridNode>, rect: QRectF) {
 
 pub enum LineNode {}
 pub fn create_line_node(s: &mut SGNode<LineNode>, size: f32, spread: f32, color: QColor) {
-    init_ressource();
+    init_resource();
     cpp!(unsafe [s as "LineNode**", size as "float", spread as "float", color as "QColor"] {
         if (!*s) *s = new LineNode(size, spread, color);
     });

--- a/examples/qmlextensionplugins/Cargo.toml
+++ b/examples/qmlextensionplugins/Cargo.toml
@@ -1,6 +1,7 @@
 [package]
 name = "qmlextensionplugins"
 version = "0.1.0"
+edition = "2018"
 authors = ["Olivier Goffart <ogoffart@woboq.com>"]
 
 [lib]
@@ -10,4 +11,3 @@ crate-type = ["dylib"]
 [dependencies]
 qmetaobject = { path = "../../qmetaobject"}
 chrono = "^0.4"
-

--- a/examples/todos/Cargo.toml
+++ b/examples/todos/Cargo.toml
@@ -1,8 +1,8 @@
 [package]
 name = "todos"
 version = "0.1.0"
-authors = ["Olivier Goffart <ogoffart@woboq.com>"]
 edition = "2018"
+authors = ["Olivier Goffart <ogoffart@woboq.com>"]
 
 [dependencies]
 qmetaobject = { path = "../../qmetaobject"}

